accepted/future-releases/class-modifiers/feature-specification.md

```diff
@@ -770,15 +771,16 @@ Define a *trivial generative constructor* to be a generative constructor that:
 
 *   declares no parameters,
 
-*   has no initializer list (no `: ...` part, so no asserts or initializers, and no super constructor invocation),
+*   has no initializer list (no `: ...` part, so no asserts or initializers, and
+    no super constructor invocation),
 
 *   has no body (only `;`), and
 
 *   is not `external`. *An `external` constructor is considered to have an
     externally provided initializer list and/or body.*
 
-A trivial constructor may be named or unnamed, and `const` or non-`const`.
-A *non-trivial generative constructor* is a generative constructor which is not a
+A trivial constructor may be named or unnamed, and `const` or non-`const`. A
```

Comment:
This seemingly trivial change is bad actually a negative for both readability and writeability.

Reflowing markdown is no more necessary than reflowing HTML, the lines will be combined when displayed anwyay, so keeping line breaks at semantically meaningful positions makes everything easier when working with the source.

(Aka: Automatic reflow is bad, m'kay!)

Reply:
I know you are logically correct, but it pains my soul to not have the lines nicely wrapped to 80 columns.

I'll undo this change.